src/chainstate/stacks/db/mod.rs

```diff
@@ -1819,4 +1820,125 @@ pub mod test {
             assert!(contract_res.is_some());
         }
     }
+
+    #[test]
+    fn test_chainstate_test_genesis_consistency() {
+        // Test root hash for the test chainstate data set
+        let mut boot_data = ChainStateBootData {
+            initial_balances: vec![],
+            first_burnchain_block_hash: BurnchainHeaderHash::zero(),
+            first_burnchain_block_height: 0,
+            first_burnchain_block_timestamp: 0,
+            post_flight_callback: None,
+            get_bulk_initial_lockups: Some(Box::new(|| {
+                Box::new(GenesisData::new(true).read_lockups().map(|item| {
+                    ChainstateAccountLockup {
+                        address: item.address,
+                        amount: item.amount,
+                        block_height: item.block_height,
+                    }
+                }))
+            })),
+            get_bulk_initial_balances: Some(Box::new(|| {
+                Box::new(GenesisData::new(true).read_balances().map(|item| {
+                    ChainstateAccountBalance {
+                        address: item.address,
+                        amount: item.amount,
+                    }
+                }))
+            })),
+        };
+
+        let path = chainstate_path("genesis-consistency-chainstate-test");
+        match fs::metadata(&path) {
+            Ok(_) => {
+                fs::remove_dir_all(&path).unwrap();
+            }
+            Err(_) => {}
+        };
+
+        let mut chainstate = StacksChainState::open_and_exec(
+            false,
+            0x80000000,
+            &path,
+            Some(&mut boot_data),
+            ExecutionCost::max_value(),
+        )
+        .unwrap()
+        .0;
+
+        let genesis_block = chainstate.clarity_state.with_marf(|marf| {
+            let index_block_hash = StacksBlockHeader::make_index_block_hash(
+                &FIRST_BURNCHAIN_CONSENSUS_HASH,
+                &FIRST_STACKS_BLOCK_HASH,
+            );
+            marf.get_root_hash_at(&index_block_hash).unwrap()
+        });
+        assert_eq!(
+            format!("{}", genesis_block),
+            "e336d28e3bccdf8e5e77216e0dea26b60292f4577b05ab2d4414776f29a5e9fc"
+        );
+    }
+
+    #[test]
+    #[ignore]
```

Comment:
Why is this `ignore`?  Does it take a long time to run?

Reply:
Yeah I had to mark this test as ignored in https://github.com/blockstack/stacks-blockchain/pull/2188/commits/5638835b9177a84964432a001dc7769d39348f6f after OOM'ing github's runner